### ISSUE TITLE
SteamDeck Unable to Press Start Game

### ISSUE THREAD

**Issue by GFreak01** [LittleCube-hax/MMRecompRando#16]
If I have the "mm_recomp_rando-0.2.3.nrm" file placed into the mods folder, pressing the "Start Game" button on the splash screen does nothing.

I've tried both the linux and AppImage releases, 
Additionally, I've tried using the Windows version through proton.  This caused the game to crash.

(This isn't an issue if I play the game on a native Windows computer)

**Comment by LittleCube-hax** [comment]
The Linux version of the mod never worked until this last release, try it and see if it helps.

**Comment by GFreak01** [comment]
I downloaded the latest release.  And I can now press the "start game" button, however after I create save file and try to open it, the game crashes.

I've tried both locally and AP hosts. Neither's log is showing any attempt to connect to the server.

I've been trying to determine if I have the  apconnect file in the wrong location, but no matter where I put it, nothing changes.
Additionally, I've tried changing "true" to "false" in the file to see if that allows me to play a non-AP file but the game still creates a new save file with 5 hearts and crashes on selecting.

**Comment by GFreak01** [comment]
In an attempt to provide more information, I launched the game via the terminal.  This is stdout.

`Failed to preload executable!
Skipping non-mod owls_never_quit-1.0.0.so
Opening mod mm_recomp_rando-0.2.3
Skipping non-mod mm_recomp_rando-0.2.3.so
Opening mod better_double_sot-0.0.1
Opening mod z_targeting_fix-1.0.0
Skipping non-mod APCpp-Glue.so
Skipping non-mod libAPCpp.so
Skipping non-mod better_double_sot-0.0.1.so
Opening mod owls_never_quit-1.0.0
Skipping non-mod z_targeting_fix-1.0.0.so
Found mods:
  mm_recomp_rando(0.2.3)
    Authors: dcvz, CelestialKitsune, VincentsSin, LittleCube
    Runtime toggleable: 0
    Dependencies: owls_never_quit:1.0.0, better_double_sot:0.0.1
  better_double_sot(0.0.1)
    Authors: danielryb
    Runtime toggleable: 0
  z_targeting_fix(1.0.0)
    Authors: danielryb
    Runtime toggleable: 0
  owls_never_quit(1.0.0)
    Authors: LittleCube
    Runtime toggleable: 0

Controller added: 0
  Instance ID: 0
SVG plugin initialised.
Loaded font face 'rmlui-debugger-font' [regular] from 'memory'.
Loaded font face 'rmlui-debugger-font' [italic] from 'memory'.
Loaded font face 'LatoLatin' [regular] from 'assets/LatoLatin-Regular.ttf'.
Loaded font face 'Chiaro' [regular] from 'assets/ChiaroNormal.otf'.
Loaded font face 'Chiaro' [bold] from 'assets/ChiaroBold.otf'.
Loaded font face 'LatoLatin' [italic] from 'assets/LatoLatin-Italic.ttf'.
Loaded font face 'LatoLatin' [bold] from 'assets/LatoLatin-Bold.ttf'.
Loaded font face 'LatoLatin' [italic, bold] from 'assets/LatoLatin-BoldItalic.ttf'.
Loaded font face 'Noto Emoji' [regular] from 'assets/NotoEmoji-Regular.ttf'.
Loaded font face 'PromptFont' [regular] from 'assets/promptfont/promptfont.ttf'.
Loading mod mm_recomp_rando
Loading mod better_double_sot
Loading mod z_targeting_fix
Loading mod owls_never_quit
save.saveInfo.playerData.healthCapacity differed
terminate called after throwing an instance of 'std::invalid_argument'
  what():  stol
Aborted (core dumped)
`
